src/components/List.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { ReactNode } from "react";
+
+interface ListProps {
+  deleteColumnButton: JSX.Element;
+  title: string;
+  children: ReactNode;
+}
+const List: React.FC<ListProps> = (props: ListProps) => {
+  return (
```

Comment:
no need to declare `children: ReactNode;` in interface ListProps
since `React.FC` includes it by default.
Also when use `Reac.FC<SomePropsInterface>` there is no need to define type for props argument.
Should be just `(props) => ...`

Reply:
src\components\List.tsx
  Line 10:14:  'deleteColumnButton' is missing in props validation  react/prop-types
  Line 11:18:  'title' is missing in props validation               react/prop-types
  Line 12:14:  'children' is missing in props validation            react/prop-types